app/assets/stylesheets/cart.css.scss

```diff
@@ -1,7 +1,9 @@
 .cart {
-  background:white;
+  background-image: 'texture.png';
```

Comment:
Properties should be ordered background, background-image<br>Prefer double-quoted strings